### DISCUSSION TITLE
Please take a quick 2 min survey on our GA plan

### DISCUSSION THREAD

**Discussion by dkershaw10** [microsoftgraph/msgraph-bicep-types#184] [Announcements]
Folks,

Please take part in a short survey to help us decide whether we are on the right track to declare GA early next year.  To get some context on what we are planning, you can watch [a video of the recent Bicep Community call](https://youtu.be/h0d8hxouT9M?t=151) where Jason walks through our remaining major features, which is also covered by the slide screenshot below:

![image](https://github.com/user-attachments/assets/e36ba5d4-c993-4634-83d1-fcb4f7401340)

Once you have that context, please take [the anonymous short survey](https://aka.ms/graphbicepsurvey)

Thanks in advance (and thanks if you have already responded on the survey). If you have any questions on the features, please let us know.

**Comment by omritc** [comment]
@dkershaw10 Is there any update on becoming GA next week?

**Comment by dkershaw10** [comment]
Hi @omritc - we're a little behind on the schedule. We've delivered all the items on the plan, except for "any client can deploy MS Graph resources with user/interactive flows".  That feature is currently rolling out, for internal validation testing, before we make it available. After that we'll fix on an updated date for GA.

**Reply by omritc** [reply]
Thank you, we are waiting waiting for that exact feature. Hopefully that's soon

**Comment by aviyam181199** [comment]
Hi @dkershaw10 ,
There's any update about the GA date?

**Comment by dkershaw10** [comment]
@aviyam181199 GA is likely towards the end of June, but could be delayed until early July.

**Reply by aviyam181199** [reply]
Hi @dkershaw10,
We’ve been tracking this since March and are really looking forward to the GA release.

Just checked again — still seems unsupported.
Any updated timeline or known blockers you can share?

Also, is there a place where we can follow progress (e.g. changelog, tracking issue, or internal planning milestone)?

Appreciate the transparency so far — just eager to get this integrated on our side.

**Comment by dkershaw10** [comment]
Due to some dependencies, it looks like GA will now be end of this month, but possibly slip into August.

The blockers that we are trying to unblock with GA are:

- any client can deploy Graph resources using the interactive flow, 
- support for Stream properties (like logo on the application resource type) and 
- hopefully CSP support via Granular Delegated Admin Permissions (GDAP)

We've also added some support for Debug mode, so you can see the Graph requests and responses for a template deployment.
